Source/WebCore/PAL/pal/PALSwift/CryptoKitShim.swift

```diff
@@ -23,273 +23,220 @@
 * THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-import CryptoKit
-import Foundation
+#if swift(>=5.9)
 
-@objc(WebCryptoErrorCodes)
-public enum ErrorCodes: Int {
-    case success = 0
-    case wrongTagSize = 1
-    case encryptionFailed = 2
-    case encryptionResultNil = 3
-    case invalidArgument = 4
-    case tooBigArguments = 5
-    case decryptionFailed = 6
-    case hashingFailed = 7
-}
+    #if SWIFTCPP_INTEROP
+        import CryptoKit
+        import Foundation
```

Comment:
Formatting nit: WebKit's style is to not have preprocessor directives (`#if`, etc.) add a level of indentation to the code. I see no reason why that should not apply to Swift code, too. So these `#if`s should have no leading whitespace and the code inside them should be normally indented.

Reply:
I am using the `Tools/Script/swift/format` for format the code.
Let me try to see if that can be modified to achieve this.

Reply:
done.

Reply:
Yeah, my understanding is that swift-format is using built-in rules, not necessarily WebKit style? I wouldn't try too hard to make it match our style, as until and unless it becomes part of `check-webkit-style` that script is only going to be run manually.